resources/src/main/java/org/robolectric/res/ResName.java

```diff
@@ -42,10 +42,11 @@ public ResName(@Nonnull String fullyQualifiedName) {
       throw new IllegalStateException("\"" + fullyQualifiedName + "\" unexpected");
   }
 
-  /**
-   * @return null if the resource could not be qualified.
-   */
-  public static String qualifyResourceName(@Nonnull String possiblyQualifiedResourceName, String defaultPackageName, String defaultType) {
+  /** Returns null if the resource could not be qualified. */
```

Comment:
You can delete it or add more information about this method's function beside current return summary, instead of converting return result summary to full summary.

Reply:
I've changed the summary according to my understand

Reply:
https://github.com/robolectric/robolectric/pull/7399/files#diff-7708adf7e87d8e9ce5a82bfef6a2ff218eb378666a887d3f36505964d81bd3c4R45